lasagne/layers/helper.py

```diff
@@ -251,7 +251,7 @@ def count_params(layer):
     """
     params = get_all_params(layer)
     shapes = [p.get_value().shape for p in params]
-    counts = [np.prod(shape) for shape in shapes]
+    counts = [np.prod(shape) if shape != () else 1 for shape in shapes]
```

Comment:
You're right. On that note it brings up another question. 

A blank numpy array is also computed as 1.